src/shared/pages/list/mpamBusinessUnits/listNames.tsx

```diff
@@ -0,0 +1,8 @@
+export const listNames = {
+    MPAM_BUS_UNITS_1UKVI: 'UKVI',
+    MPAM_BUS_UNITS_2: 'BF',
+    MPAM_BUS_UNITS_3: 'IE',
+    MPAM_BUS_UNITS_4: 'EUSS',
+    MPAM_BUS_UNITS_5: 'HMPO',
+    MPAM_BUS_UNITS_6: 'Windrush',
+    MPAM_BUS_UNITS_7: 'Corona Virus' };
```

Comment:
Is this the correct stylisation of coronavirus/corona virus?

Reply:
No, I'll update to match elsewhere (Coronavirus (COVID-19))